### DISCUSSION TITLE
How to custom the menu?

### DISCUSSION THREAD

**Discussion by xujinglong2021** [tldraw/tldraw-v1#545] [Q&A]
I'm a beginner. I have read the guide (https://github.com/tldraw/tldraw/blob/main/guides/documentation.md). I have to say it's a little help for me. but I would like to know if there is a handbook with more detail about development.
I'm a beginner. I have read the guide (https://github.com/tldraw/tldraw/blob/main/guides/documentation.md). I have to say it's a little help for me. but I would like to know if there is a handbook with more detail about development.

**Comment by steveruizok** [comment]
At the moment you can't customize the menu apart from forking the repository. We do however have an API for customizing menus in the new version. We'll have docs for that head of our open source release!
